packages/api/src/store/table.ts

```diff
@@ -371,15 +371,6 @@ export default class Table<T extends DBObject> {
 
   // on startup: auto-create indices if they don't exist
   async ensureIndex(propName: string, prop: FieldSpec, parents: string[] = []) {
-    if (
-      process.env.NODE_ENV !== "test" &&
-      ["stream", "session"].includes(this.name)
```

Comment:
Btw if you want to be extra sure that the indexes are being created normally you can also create a test [like this](https://github.com/livepeer/livepeer-com/blob/4e24f8e656e204089b0d0057428820045c162c89/packages/api/src/store/asset-table.test.ts#L8) for the `stream-table`

Reply:
Ok. I'll probably not add the test, because to check it would not only need to check if the index exists, but also its content. So, probably not worth adding it.